res/values-v14/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+
+    <dimen name="widget_margin">0dp</dimen>
```

Comment:
But apart from that the whole alignment of the widget is still off. :-) I will have to see how to do that properly.